fvm/src/lib.rs

```diff
@@ -96,7 +101,8 @@ mod test {
             _h2: &[u8],
             _extra: &[u8],
         ) -> anyhow::Result<(Option<fvm_shared::consensus::ConsensusFault>, i64)> {
-            todo!()
+            // consensus is always valid for tests :)
+            anyhow::Result::Ok((None, 0))
```

Comment:
```suggestion
            Ok((None, 0))
```

You only need to use `anyhow::Result::` in a closure where you want rust to guess the return type. Otherwise, you can just use `Ok` and rust will figure out the error type.

Reply:
Rust at one point was having issues figuring the type out, though it isn't necessary anymore.